share/logcfg.dat

```diff
@@ -129,6 +129,8 @@ CWTONE=800
 #           for ten tec OMNI VI #
 #################################
 #
+#SYNC_RIG_MODE
```

Comment:
Maybe we should do it as
`#SYNC_RIG_MODE=OFF`

`ON` is default state. So why should we remove the leading `#` to just turn it on a second time.

Reply:
A second though: How about calling it RIG_SYNC_MODE just to have it similar to other rig related commends.

Reply:
> A second though: How about calling it RIG_SYNC_MODE just to have it similar to other rig related commends.

Agree (see my [previous](https://github.com/Tlf/tlf/pull/458#discussion_r1925862247) comment).

Reply:
How about `RIG_MODE_SYNC` ? As it is more about synchronizing rig mode than the mode of "rig sync" as `RIG_SYNC_MODE` would suggest.

Reply:
Both sound good for me.